src/js/irc-utils.js

```diff
@@ -19,17 +17,14 @@ IrcUtils.service('IrcUtils', [function() {
      };
 
     /**
-     * Get a new version of a nick list, sorted by last speaker
+     * Get a new version of a nick list
      *
      * @param nickList Original nick list
-     * @return Sorted nick list
+     * @return list of nick names
      */
     var _ciNickList = function(nickList) {
```

Comment:
this is what's used for nick completion, are you sure we don't want to keep the sorting here?

Reply:
we could do `nickList.sort(sortBy('spokeAt')).map((el) => el.name).reverse()`?

Reply:
Ah, it's already filled with a sorted list (but we still need to reverse?)